EIPS/eip-hard-gas-target.md

```diff
@@ -0,0 +1,60 @@
+---
+eip: xxxx
+title: Hardcoded Block Gas Target
+author: Philippe Castonguay (@PhABC)
+discussions-to: xxxx
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-03-13
+
+---
+
+## Simple Summary
+
+Hardcode the block gas target (previously known as *block gas limit* before [EIP-1559](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1559.md)) to `12,500,000` gas per block.
+
+## Abstract
+
+Ethereum's block gas target is currently dictated by block producers and is not enforced when validating blocks in clients. This EIP proposes to hardcode the block gas target to `12,500,000`.
+
+## Motivation
+
+Both Ethereum's Proof of Work and Proof of Stake designs assume that block producers are financially rational, but does not assume block producers to be benevolent. There is one exception however, and it is when block producers choose the gas targe of a block (previously known as block gas limit), where it is assumed that block producers care about the long term health and decentralisation of the chain. Indeed, the block gas target is one of the only parameters in Ethereum that is not dictated by node consensus, but instead is chosen by block producers. This decision was initially made to allow urgent changes in the block gas target if necessary. Both drastically increasing or decreasing this parameter could have serious consequences that may not be desired. It is therefore a critical parameter that should require node consensus to avoid any sudden harmful change imposed by a small number of actors on the rest of the network.
+
+## Specification
+This EIP assumes that EIP-1559 is implemented and blocks' `gasLimit` was replaced with `gasTarget`.
+
+#### Block Header
+
+As of `FORK_BLOCK_NUMBER`, remove `gas_target` field from block headers.
+
+#### Consensus
+
+As of `FORK_BLOCK_NUMBER`, change the headers gas target validity check such that `header.gasUsed` **MUST** be smaller than `BLOCK_GAS_TARGET * ELASTICITY_MULTIPLIER` where `BLOCK_GAS_LIMIT` is a hard-coded constant set to `12,500,000`, not the current gas target included in block headers and `ELASTICITY_MULTIPLIER` is `2` as per [EIP-1559 specifications](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1559.md#specification).
+
+#### EVM
```

Comment:
I don't think this needs to be specified as it is implied.

Reply:
I'll leave it to be explicit about the `<` instead of `<=`, which could cause a fork.

Reply:
I was actually referring to the EVM section here. I realize now that if the header *didn't* have `gas_limit` specified, this would be needed. However, if the header does include the `gas_limit` the result of `GASLIMIT` should be implied.

Reply:
> I'll leave it to be explicit about the < instead of <=, which could cause a fork if not implemented the same.

What is the rationale for `<` vs. `<=`? I believe today `gas_used` must be less than or equal to `gas_limit`, seems like it is better to maintain that unless there is a particular reason to change?